Feature footer

anhfactor opened the pull request:
Done!
<img width="1426" alt="Screen Shot 2023-01-11 at 23 51 03" src="https://user-images.githubusercontent.com/13186215/211867836-72e22873-2e96-43cf-a0f7-4719a58d3543.png">

anhfactor committed:
feature footer

anhfactor committed:
run rush update

DistractionBoy commented on libs/ui/src/molecules/Footer/Footer.tsx:
```diff
@@ -0,0 +1,112 @@
+                    Discover, collect, and sell digital items on the
+                </div>
+                <div className="rounded-md text-sm font-medium text-white">
+                  largest NFT marketplace on Optimism
```
could we change the text to say, "largest Layer 2 NFT marketplace"?

DistractionBoy commented on libs/ui/src/molecules/Footer/Footer.tsx:
```diff
@@ -0,0 +1,112 @@
+              </div>
+            </div>
+            <div className="flex flex-1 justify-center lg:justify-end">
+              <div className="lg:mr-20">
```
i know in the design, the right side doesn't exactly line up with the rest of the footer content below it, but check with Daru because i'm not sure if that was intentional or not. we might want to take out the mr-20 class

DistractionBoy commented on libs/ui/src/molecules/Footer/Footer.tsx:
```diff
@@ -0,0 +1,112 @@
+                      type="button"
+                      className="flex-shrink-0 px-2 py-2 rounded-lg bg-sky-900"
+                    >
+                  <FaTelegram className="h-6 w-6 text-white"/>
```
it looks like the design has more padding / larger button total size (50px). could you adjust the sizes a little? also check the background color

anhfactor committed:
- change text to "largest Layer 2 NFT marketplace"
- add image background cover for footer
- change button social color
- adjust footer for md & lg screen

anhfactor committed:
adjust footer for mobile & laptop

anhfactor committed:
reduce border height <hr> for mobile

anhfactor commented:
- Mobile version:
<img width="282" alt="Screen Shot 2023-01-15 at 00 30 53" src="https://user-images.githubusercontent.com/13186215/212486997-ee3cbc94-5e1e-4fda-a2a3-1395e4f34b23.png">

- Laptop version: 
<img width="1427" alt="Screen Shot 2023-01-15 at 00 24 46" src="https://user-images.githubusercontent.com/13186215/212486784-68af7478-62d2-4473-bcc8-8a3eb5e6720d.png">

DistractionBoy reviewed (approved):
LGTM
